tests/resources/VCH-Util.robot

```diff
@@ -322,7 +322,8 @@ Run Secret VIC Machine Inspect Command
 
 Run VIC Machine Delete Command
     ${rc}  ${output}=  Run Secret VIC Machine Delete Command  %{VCH-NAME}
-    Wait Until Keyword Succeeds  6x  5s  Check Delete Success  %{VCH-NAME}
+    ${status}=  Run Keyword And Return Status  Wait Until Keyword Succeeds  6x  5s  Check Delete Success  %{VCH-NAME}
+    Run Keyword Unless  ${status}  Fatal Error  vic-machine delete has failed to remove the VCH: %{VCH-NAME} as expected
```

Comment:
Is there any way we could have a suite/group cleanup check?
In both the vic-machine-service tests and the delete-resource-pool tests we're leaking because we don't call vic-machine delete, not because it fails.